src/visualization/src/IrrlichtUtils.h

```diff
@@ -73,7 +66,25 @@ inline iDynTree::Rotation irr2idyntree_rot(const irr::core::vector3df & rotIrr)
 
 inline const irr::core::vector3df idyntree2irr_rot(const iDynTree::Rotation & rot)
 {
-    return idyntree2irr_rpy(rot.asRPY());
+    irr::core::matrix4 irrTransform(irr::core::matrix4::EM4CONST_IDENTITY);
+    //Transposing the matrix since irrlicht uses left handed frames
```

Comment:
Can you elaborate a bit more on this? I thought that the rendering problem of STL meshes was solved by changing the scale of the X variable, instead also this is necessary? Did you tested this with the iCub 2.5 DAE models after this changes?

Reply:
I reverted the changes in https://github.com/robotology/idyntree/pull/802/commits/0a6f821fed94b0ae34a1ce8c747c33fa5a06b116. 
Regarding the visualization, I run ``idyntree-model-view`` with:
- the `iCubGenova09`` urdf
![image](https://user-images.githubusercontent.com/18591940/105531643-ec81d880-5ce9-11eb-8a00-5eb5e22a8bde.png)
- the ``iCubGenova04`` urdf
![image](https://user-images.githubusercontent.com/18591940/105531690-ff94a880-5ce9-11eb-982c-712f90264915.png)
- a custom ``iCubGenova04`` urdf with dae meshes
![image](https://user-images.githubusercontent.com/18591940/105531736-13d8a580-5cea-11eb-81c5-c3811376fd28.png)

Reply:
Ok thanks! However, if I recall correctly is that the problem with the stl meshes on iCub3 (the one that we imagine was fixed by the STL load fix) only appeared when the legs or arm were moved from the zero position, so after you reverted the change to the rotation did you check if the visualization is still working as expected?

Reply:
Here the result when reading the joints from the control boards:
- ``iCubGenova09`` urdf
![image](https://user-images.githubusercontent.com/18591940/105829099-386f9e80-5fc4-11eb-8c23-55b62b80ec51.png)
- ``iCubGenova04`` urdf
![image](https://user-images.githubusercontent.com/18591940/105829371-9603eb00-5fc4-11eb-819c-9fb39ea34dad.png)
- ``iCubGenova04`` urdf with custom DAE meshes
![image](https://user-images.githubusercontent.com/18591940/105829537-c8154d00-5fc4-11eb-8488-8718458c5a27.png)

Reply:
Great!